.circleci/config.yml

```diff
@@ -21,6 +11,12 @@ anchor_test_python: &anchor_test_python
     # https://github.com/googleapis/gapic-generator/issues/2208
   when: always
 
+anchor_test_ruby: &anchor_test_ruby
+  command: |
+    cd workspace/gapic-generator/artman-genfiles/ruby/google-cloud-ruby/${TEST_API}
+    bundle update
+    bundle exec rake test
```

Comment:
Did we also want smoke tests?

Reply:
`bundle exec rake acceptance`

https://github.com/googleapis/gapic-generator/blob/master/src/test/java/com/google/api/codegen/testdata/ruby/ruby_library.baseline#L414